src/ResourceManager/LocationBasedServices/Commands.LocationBasedServices/LocationBasedServicesAccount/GetAzureLocationBasedServicesAccount.cs

```diff
@@ -0,0 +1,121 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.LocationBasedServices.Models;
+using Microsoft.Azure.Management.LocationBasedServices;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Rest.Azure;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.LocationBasedServices
+{
+    /// <summary>
+    /// Get Location Based Services Account by name, all accounts under resource group or all accounts under the subscription
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, LocationBasedServicesAccountNounStr), OutputType(typeof(PSLocationBasedServicesAccount))]
```

Comment:
Need a DefaultParameterSetName, since there is more than one parameter set

Reply:
ok.  does it matter which one?  i picked.